Filling database with entries

NirmalManoj opened the pull request:
I have made entries for all tables except some in pages. I have made a page for a company and a page for entertainment: we need to make more pages for other sub-classes of PAGE. 

I have checked what I have added. So most likely, it will work in yours too. Before approving, check if I covered all the tables except those under PAGE.

NirmalManoj committed:
USER, POST, PROFILE, EDUCATION DUMMY CREATED

NirmalManoj committed:
Dummy data: COMMENT, COMMENTS, FOLLOWS, REACTS, MENTIONS, IS_TAGGED

NirmalManoj committed:
MESSAGE, GROUP, BELONGS_TO, IS_ADMIN, IS_MODERATOR, SENDS_SPECIFIC/Gen

and SHARES

NirmalManoj committed:
STORIES and RESPONDS

NirmalManoj committed:
PAGES: Company and Entertainment

alapan-sau reviewed (approved):
Tested! Works Fine.
Add some more dummy data if possible! that can be however done later :)
Also Importantly, can we put some filters like --
1. If we make a user an admin/moderator, we need to check if he is in the group first?
I implemented the same in delete-- as in, if a user is removed from a group , he  is removed from the admin/moderator also.

NirmalManoj commented:
> Tested! Works Fine.
> Add some more dummy data if possible! that can be however done later :)
> Also Importantly, can we put some filters like --
> 1. If we make a user an admin/moderator, we need to check if he is in the group first?
> I implemented the same in delete-- as in, if a user is removed from a group , he  is removed from the admin/moderator also.

Yes. I had implemented the first filter - yesterday itself, before sleeping. ~~We have to take care of the second too.~~
